docs/source/conf.py

```diff
@@ -223,3 +224,8 @@
     shutil.copytree('../../examples', './examples')
 except FileExistsError:
     pass
+
+# redirects
+redirects = {
```

Comment:
I found a solution for this. It turns out redirects don't work if a file, from which to redirect, doesn't exist. So let's create an empty file from which to redirect from:

```Python
# redirects
redirects = {
    'examples/processing_api_request': 'process_request.html',
}
# create empty files to redirect from
for file_path in redirects:
    file_path = f'{file_path}.rst'
    open(file_path, 'w').close()
```
I can confirm this works. Perhaps in the future, they will improve this...

Reply:
👍